### PR TITLE
Enable dhcp on ironicendpoint interface for v1alpha3

### PR DESCRIPTION
**Pull request by fmuyassarov**
When v1alpha3, enable DHCP for ironicendpoint interface.

### CODE REVIEW

File: vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc [hunk 1]
```diff
@@ -145,8 +145,12 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
             bridges:
               {{ IRONIC_ENDPOINT_BRIDGE }}:
                 interfaces: [enp1s0]
-                addresses: 
-                - {{ "{{ ds.meta_data.provisioningIP }}" }}/{{ "{{ ds.meta_data.provisioningCIDR }}" }}  
+{% if CAPI_VERSION == "v1alpha3" %}
+                  dhcp4: true
```
Comment (maelk) [inline vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc:149]: the indentation is wrong here

File: vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc [hunk 2]
```diff
@@ -145,8 +145,12 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
...
+{% else %}
+                  addresses:
+                  - {{ "{{ ds.meta_data.provisioningIP }}" }}/{{ "{{ ds.meta_data.provisioningCIDR }}" }}
```
Comment (maelk) [inline vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc:152]: same here

Comment (fmuyassarov) [reply vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc]: Fixed.

File: vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc (hunk 1 above)
Comment (fmuyassarov) [reply vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc]: good catch, fixed, thank you.

**Comment by jan-est** [comment]
lgtm

**Pull request merged**